### PR TITLE
[patch] Add roletoken option and force enable

### PR DESCRIPTION
**Pull request by kyfujisa**
Added an option to force roletoken to be enable.

### CODE REVIEW

File: usecase/authz_proxyd.go [hunk 1]
```diff
@@ -191,6 +193,19 @@ func newAuthzD(cfg config.Config) (service.Authorizationd, error) {
 				),
 			),
 		}
+	} else {
```
Comment (kyfujisa) [inline usecase/authz_proxyd.go:203]: In athenz-authorizer, access token is enabled by default.
https://github.com/yahoojapan/athenz-authorizer/blob/d896d3edec8f0d2abd90d20ff3ffd84abb2e702c/option.go#L38
Therefore, the existing code always set access token to enable.
Added an `else` to the `if` to make it properly disable.
The conditions of the JWKD are also unified here.

File: config/config_test.go [hunk 2]
```diff
@@ -123,6 +123,81 @@ func TestNew(t *testing.T) {
...
+						CertBackdateDur: "1h",
+						CertOffsetDur:   "1h",
+					},
+				},
+			},
+		},
+		{
+			name: "Test no roletoken",
+			args: args{
+				path: "./testdata/undefine_roletoken.yaml",
```
Comment (kyfujisa) [inline config/config_test.go:146]: Load a config not written by roleotken and test that it is `Role.Enable: true`.

**Comment by kyfujisa** [comment]
Regarding the fix for de0f38e.
The change of access token to disable causes `error no verifiers` error.
I added either Role or Access to the test case to make sure this doesn't happen.

**Comment by WindzCUHK** [comment]
LGTM

**Pull request merged**